package/common_init/overlay/etc/init.d/S80endpoint_router_sbp

```diff
@@ -11,12 +11,15 @@ setup_permissions()
 }
 
 ins_output_mode=$(query_config --section ins --key output_mode)
-starling_daemon_enabled=$(query_config --section system --key enable_starling_daemon)
+starling_daemon_enabled=$(query_config --section experimental_flags --key starling_on_linux)
+
+echo "0" > /persistent/flags/starling_on_linux
```

Comment:
random thought, not sure the endpoint router init script is the most logical place to init this. but if not here, where? maybe another init script that has default values for these which would launch after `qspi_persistent` but before `endpoint_router_sbp` or any other dependent scripts? anyways, not critical

Reply:
I looked for one as well: 
I was looking for a central script that configured all of these more "permanent" settings.
However, it seems that `starling_on_linux` is the first of these kinds of flags. 
Creating a separate init script for just setting this flag seemed a bit overkill, until we have a few more settings we want to manage